src/components/custom/_dhero-2.js

```diff
@@ -189,6 +189,12 @@ const query = graphql`
         dmt5_mobile: file(relativePath: { eq: "dmt5_trade_mobile.png" }) {
             ...fadeIn
         }
+        derivx: file(relativePath: { eq: "derivx/hero-laptop.png" }) {
```

Comment:
Could we name the image folder and also the page folder to `deriv-x` maybe? 😬 People might think it was a typo otherwise.